openspec/changes/archive/2026-03-06-feat-cli-score-display/specs/core-engine/spec.md

```
@@ -0,0 +1,29 @@
+## MODIFIED Requirements
+
+### Requirement: Two-Stage Retrieval
+The core engine SHALL provide scout and inspect operations for efficient context retrieval with support for chunk-level results and parent document context. 核心引擎 SHALL 提供 scout 和 inspect 操作以进行高效的上下文检索，支持块级结果和父文档上下文。
+
+#### Scenario: 侦察相关的文档和块
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)